src/map/packets/quest_mission_log.cpp

```diff
@@ -31,248 +31,262 @@
+        if (status == 0x01) {
+            generateCurrentMissionPacket(PChar);
+            logType = MISS_CURRENT;
+            break;
+        }
+        if (status == 0x02) {
+            generateCompleteMissionPacket(PChar);
+            logType = MISS_COMPLETE;
+            break;
+        }
+    case MISSION_WINDURST:
+        if (status == 0x01) {
+            generateCurrentMissionPacket(PChar);
+            logType = MISS_CURRENT;
+            break;
+        }
+        if (status == 0x02) {
+            generateCompleteMissionPacket(PChar);
+            logType = MISS_COMPLETE;
+            break;
+        }
 
-		case MISSION_ZILART:
-			if(status == 0x01) {
-				generateCurrentMissionPacket(PChar);
-				logType = MISS_CURRENT;
-				break;
-			}
-			if(status == 0x02) {
-				generateCompleteMissionPacket(PChar);
-				logType = MISS_COMPLETE;
-				break;
-			}
-		case MISSION_COP:
-		case MISSION_CRISTALLINE_PROPHECY:
-		case MISSION_MOOGLE_KUPO_DETAT:
-		case MISSION_SHANTOTTO_ASCENSION:
-		case MISSION_ADOULIN:
-        case MISSION_RHAPSODIES:
-			if(status == 0x01)
-			{
-				generateCurrentMissionPacket(PChar);
-				logType = MISS_CURRENT;
-				break;
-			}
-	}
+    case MISSION_ZILART:
+        if (status == 0x01) {
+            generateCurrentMissionPacket(PChar);
+            logType = MISS_CURRENT;
+            break;
+        }
+        if (status == 0x02) {
+            generateCompleteMissionPacket(PChar);
+            logType = MISS_COMPLETE;
+            break;
+        }
+    case MISSION_COP:
+    case MISSION_CRISTALLINE_PROPHECY:
+    case MISSION_MOOGLE_KUPO_DETAT:
+    case MISSION_SHANTOTTO_ASCENSION:
+    case MISSION_ADOULIN:
+    case MISSION_RHAPSODIES:
+        if (status == 0x01)
+        {
+            generateCurrentMissionPacket(PChar);
+            logType = MISS_CURRENT;
+            break;
+        }
+    }
 
-	WBUFW(data,(0x24)) = logType;
+    WBUFW(data, (0x24)) = logType;
 }
 
 void CQuestMissionLogPacket::generateQuestPacket(CCharEntity * PChar, uint8 logID, uint8 status)
 {
-	if(status == 0x01)
-		memcpy(data + 4, PChar->m_questLog[logID].current, 32);
-	else if(status == 0x02)
-		memcpy(data + 4, PChar->m_questLog[logID].complete, 32);
+    if (status == 0x01)
+        memcpy(data + 4, PChar->m_questLog[logID].current, 32);
+    else if (status == 0x02)
+        memcpy(data + 4, PChar->m_questLog[logID].complete, 32);
 }
 
 void CQuestMissionLogPacket::generateCurrentMissionPacket(CCharEntity * PChar)
 {
-	uint16 add_on_scenarios = 0;
+    uint16 add_on_scenarios = 0;
 
-	add_on_scenarios +=	PChar->m_missionLog[9].current;
-	add_on_scenarios +=	PChar->m_missionLog[10].current << 0x04;
-	add_on_scenarios +=	PChar->m_missionLog[11].current << 0x08;
-	// Not perfect, but they display and missions DO progress. Can fix properly later. There is a delay before when the menu updates. Zoning will force it.
+    add_on_scenarios += PChar->m_missionLog[MISSION_CRISTALLINE_PROPHECY - 12].current;
```

Comment:
i realise you didnt make the typo, but could you change this to MISSION_CRYSTALLINE_PROPHECY instead